cross-build: PR 58174 causing cross build failure

```
W0131 01:55:36.939] # k8s.io/kubernetes/pkg/kubelet/dockershim
W0131 01:55:36.940] pkg/kubelet/dockershim/helpers_windows.go:52:3: undefined: modifyHostNetworkOptionForContainer
```

https://github.com/kubernetes/kubernetes/pull/58174
https://k8s-testgrid.appspot.com/sig-release-misc#cross-build&graph-metrics=test-duration-minutes
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-cross-build/4469

---
cc: @kubernetes/sig-node-bugs @kubernetes/sig-node-test-failures @filbranden @Random-Liu @vishh
/kind bug
/priority failing-test

--- a/pkg/kubelet/dockershim/helpers_windows.go
+++ b/pkg/kubelet/dockershim/helpers_windows.go
@@ -49,7 +49,7 @@ func (ds *dockerService) updateCreateConfig(
 		createConfig.HostConfig.NetworkMode = dockercontainer.NetworkMode(networkMode)
 	} else {
 		// Todo: Refactor this call in future for calling methods directly in security_context.go
-		modifyHostNetworkOptionForContainer(false, podSandboxID, createConfig.HostConfig)
+		modifyHostOptionsForContainer(false, podSandboxID, createConfig.HostConfig)
 	}
 
 	return nil

